frontend/src/scenes/funnels/funnelLogic.js

```diff
@@ -0,0 +1,72 @@
+import { kea } from 'kea'
+import api from 'lib/api'
+import { toParams } from 'lib/utils';
+
+
+
+export const funnelLogic = kea({
+    key: props => props.id,
+    actions: () => ({
+        setFilters: input => input
```

Comment:
I suggest doing `setFilters: filters => ({ filters })` instead. The actions are actually better called "action function argument serialisers", meaning that if in the React side you call `logic.actions.setFilters(filters)`, then this "action" will serialize the function argument into a payload object.